server/src/lib/model/user-client.ts

```diff
@@ -437,6 +530,7 @@ const UserClient = {
       updateDemographicsPromise,
       this.claimContributions(clientId, clientIds),
       languages && updateLanguages(clientId, languages),
```

Comment:
Wondering here instead of adding another line for variants we rename `updateLanguages` to `updateAccents` and then add a function like:

```js
/**
 * Updates languages for a given user
 *
 * @param clientId clientId as string
 * @param languages array of UserLanguage
 * @returns void
 */
async function updateLanguages(clientId: string, languages: UserLanguage[]) {
  return Promise.all([
    updateAccents(clientId, languages),
    updateVariants(clientId, languages),
  ]);
}
```